splines/SplineCommon.h

```diff
@@ -12,9 +12,9 @@
 #endif
 
 /// KS: We store coefficients {y,b,c,d} in one array one by one, this is only to define it once rather then insert "4" all over the code
-#define _nCoeff_ 4
+constexpr inline ushort nCoeff = 4;
 /// KS: For TF1 we store at most 5 coefficients, we could make it more flexible but for now define it here to make future changes easier to track
-#define _nTF1Coeff_ 2
+constexpr inline ushort nTF1Coeff = 2;
```

Comment:
Could you revert this to define?
These two are used in GPU.
https://github.com/mach3-software/MaCh3/blob/develop/splines/gpuSplineUtils.cu

I don't know but this could cause problem on older gpu. Updating GPU is harder than gcc. GPU conianters are pain.
No prefence for name

Reply:
yep, would be good to check what nvcc versions are in use and when they support features like this, but if its needed for gpu to be a macro, thats best